src/mame/drivers/nes_clone.cpp

```diff
@@ -397,6 +458,142 @@ void nes_clone_state::nes_clone_pal(machine_config &config)
 	m_maincpu->add_route(ALL_OUTPUTS, "mono", 0.50);
 }
 
+/**************************************************
+ Dancing Expert specifics
+**************************************************/
+
+void nes_clone_dancexpt_state::machine_start()
+{
+	nes_clone_state::machine_start();
+	m_ppu->space(AS_PROGRAM).install_read_bank(0x0000, 0x1fff, m_bank_gfx);
+	m_bank_gfx->set_base(memregion("gfx1")->base());
+
+	m_nt_ram.resize(0x800);
```

Comment:
Please use `unique_ptr` rather than `vector` for things that are allocated once and don’t need to be resized.